src/components/UserStats/IncomeDetailsDialog.vue

```diff
@@ -0,0 +1,186 @@
+<template>
+  <Dialog
+    :open="isOpenDialog"
+    panel-container-class="min-w-[300px] max-w-[90vw] laptop: max-w-[720px]"
+    panel-component="CardV2"
+    panel-class="overflow-y-scroll shadow-lg"
+    @close="$emit('close')"
+  >
+    <div v-if="isLoading" class="flex justify-center my-[48px]">
+      <ProgressIndicator v-if="isLoading" />
+    </div>
+    <div v-else class="flex flex-col gap-[16px]">
+      <UserStatsIncomeMenuItem
+        :current-target-type="currentTargetType"
+        :type-menu-item-list="typeMenuItemList"
+        @on-type-click="handleTypeClick"
+      />
+      <UserStatsIncomeTable
+        :current-target-type="currentTargetType"
+        :current-list="currentList"
+      />
+    </div>
+  </Dialog>
+</template>
+
+<script>
+import { ellipsis } from '~/util/ui';
+import { convertToLIKEPrice } from '~/util/nft';
+import nftMixin from '~/mixins/nft';
+import { logTrackerEvent } from '~/util/EventLogger';
+
```

Comment:
Just sorting import that group by subfolder
```suggestion
import nftMixin from '~/mixins/nft';

import { logTrackerEvent } from '~/util/EventLogger';
import { convertToLIKEPrice } from '~/util/nft';
import { ellipsis } from '~/util/ui';
```